main.go

```diff
@@ -71,6 +71,11 @@ func main() {
 		setupLog.Error(err, "unable to initialize controller with chart renderer")
 		os.Exit(1)
 	}
+	ec, err = ec.InitializeControllerWithImageVector()
+	if err != nil {
+		setupLog.Error(err, "unable to initialize controller with image vector")
+		os.Exit(1)
+	}
```

Comment:
Will you please replace line65-to line74 with single call to `controller.NewReconcilerWithImageVector `?

Reply:
done